LayoutTests/fast/css/default-bidi-css-rules-expected.txt

```diff
@@ -1,15 +1,15 @@
 This test checks the default rules for direction and unicode-bidi CSS properties.
 
 PASS styleOf("div", {}).direction is "ltr"
-FAIL styleOf("div", {}).unicodeBidi should be isolate. Was normal.
+PASS styleOf("div", {}).unicodeBidi is "isolate"
```

Comment:
Do you know if all of these are covered by WPT? Maybe we could begin deleting some of these `fast/css` test cases where they are covered by WPT. (Note: This is an idea for a follow-up patch, not for today's patch).

Reply:
Yes - I also think now that we have WPT, we should get rid of it.